content/usage/services.md

```diff
@@ -94,6 +94,10 @@ compose:
 
 For security reasons this option is only available to trusted repositories. Trusted repositories are enabled on a per-repository basis by a Drone administrator from your repository settings screen.
 
+Alternatively, each service will have access to the current source code under the ```/drone``` directory
```

Comment:
I could see value in providing at least one example after the ``<repository>`` line, as ``<repository>`` won't be immediately obvious to everyone. I tend to really appreciate seeing pattern strings spelled out at least once so I can feel more confident before I start.